spectacle.cabal

```diff
@@ -41,10 +42,17 @@ common common
     -Wredundant-constraints
     -fshow-warning-groups
     -Wmissing-import-lists
-    -Wunused-packages
 
   build-depends:
-      base >=4.14 && <4.15
+      base           >= 4.14 && < 4.17
```

Comment:
If GHC `9.4.2` is to be supported then the bounds on `base` need to be `base           >= 4.14 && < 4.18`.